src/components/PostDetails.tsx

```diff
@@ -1,117 +1,156 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Post } from '../types/Post';
 import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
+import { Comment } from '../types/Comment';
+import { client } from '../utils/fetchClient';
+
+type Props = {
+  targetPost: Post;
+};
+
+export const PostDetails: React.FC<Props> = ({
+  targetPost,
+}) => {
+  const [comments, setComments] = useState<Comment[]>([]);
+  const [showError, setShowError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isOpenForm, setIsOpenForm] = useState(false);
+
+  const [isAdding, setIsAdding] = useState(false);
+
+  async function getComments(postId: number) {
+    setIsLoading(true);
+    const url = `/comments?postId=${postId}`;
+
+    try {
+      const allComments: Comment[] = await client.get(url);
+
+      setComments(allComments);
+    } catch (error) {
+      setShowError(true);
+      // eslint-disable-next-line no-console
+      console.log(error);
```

Comment:
not fixed 
![image](https://user-images.githubusercontent.com/70009205/210410916-cb4f8c81-a6df-4e33-80d5-d1af3d5fdce9.png)
If you have a comment about something, always check your entire code and make corrections everywhere